app/web/features/messages/requests/HostRequestView.tsx

```diff
@@ -154,6 +156,40 @@ export default function HostRequestView({
 
   const handleBack = () => router.back();
 
+  const renderUserSummary = () => (
```

Comment:
A minor thing - since you don't need to pass anything to the function, did you know you can assign the JSX directly to the variable, so then you can reference it directly (as opposed to needing to call it when assigned to a function)?

Reply:
Ooo I actually did not know that, but it makes sense now that you said it! You're right I guess there's nothing in there that's a state change that wouldn't just be a page reload.

Reply:
It makes more sense if you think of it as (since JSX is syntactic sugar of the below!):
```typescript
const userSummary = React.createElement(...)
```

otherwise intuitively it's like "why am I allowed to assign HTML to JS variables?" 😄

Reply:
Nah totally fare I was thinking it as I did it but was like gah there's already a "UserSummary" so I named it "userSummarySection". Done!